src/test/java/com/iexec/common/utils/FeignBuilderTest.java

```diff
@@ -37,6 +38,14 @@ void stopServer() {
         mockServer.stop();
     }
 
+    @Test
+    void testBuilderCreation() {
+        assertNotNull(FeignBuilder.createBuilder(Logger.Level.FULL)
+                .target(FeignTestClient.class, "http://localhost:" + mockServer.getPort()));
+        assertNotNull(FeignBuilder.createBuilderWithBasicAuth(Logger.Level.FULL, "username", "password")
+                .target(FeignTestClient.class, "http://localhost:" + mockServer.getPort()));
+    }
+
```

Comment:
In the same way you added:
```
@Test
void testJsonDecoder()
```
Why not adding/update it to parse a Duration field?

Reply:
It won't work this way. A mock test will never execute the jsr310 library code and will provide a false sense of security.
If we want to add a test, it must be implemented differently. It must not be a mock test.

Reply:
done, see a64bb5